assets/js/blocks/featured-product/block.js

```diff
@@ -276,8 +276,9 @@ const FeaturedProduct = ( {
 			style.backgroundColor = overlayColor.color;
 		}
 		if ( focalPoint ) {
-			style.backgroundPosition = `${ focalPoint.x *
-				100 }% ${ focalPoint.y * 100 }%`;
+			const bgPosX = focalPoint.x * 100;
+			const bgPosY = focalPoint.y * 100;
+			style.backgroundPosition = `${ bgPosX }% ${ bgPosY }%`;
```

Comment:
I see you slipped this in here 😄 . Are there any other places where similar code might exist that should be done in a follow-up?

Reply:
I saw a couple of other template literals which get broken: [#](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/953/files#diff-a0ee0e38ae0fb1b91f54d4d1c774d7b5R180), [#](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/953/files#diff-5907cb415f8589a14d14f478d5d470edR81). But they don't have any operation inside, so my guess is that [with the new version of Prettier they will no longer break](https://prettier.io/blog/2019/06/06/1.18.0.html#stop-breaking-simple-template-literals-5979-by-jwbay), so I would leave them like that and let it auto-fix once `wp-prettier` is updated.